all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -109,3 +118,20 @@ items:
     clusterIP: None
     selector:
       jaeger-infra: jaeger-pod
+- apiVersion: v1
```

Comment:
What is the benefit of defining it as a separate service?

Reply:
It makes it a simple replacement for zipkin in istio without having to change the istio.yaml